webapi/Controllers/ChatMemoryController.cs

```diff
@@ -89,7 +88,7 @@ public class ChatMemoryController : ControllerBase
             // Search if there is already a memory item that has a high similarity score with the new item.
             var filter = new MemoryFilter();
             filter.ByTag("chatid", chatId);
-            filter.ByTag("memory", sanitizedMemoryName);
+            filter.ByTag("memory", memoryName);
```

Comment:
Although probably out of the scope of this change, you might want to rename all instances of "memoryName" in all of our code to "memoryType".  If you don't do it, I will at one point.  This has been annoying me for a while.  Feel free to ignore for now.  As I said, I'll get around to it if you don't.

Reply:
I'll go look for places that need to be renamed. But keep in mind that there are still uses of memoryName when it comes to accessing memory - memoryType is just the public contract for the API so that things don't break when the memory name in appsettings.json is changed.

Reply:
I guess what I'm saying is that the type of memory we use should never have been named MemoryName but rather MemoryType.  It's confusing.  This PR is a case in point,

Reply:
Yes I agree, it's super confusing. I'm renaming some of it to `memoryContainerName` to try to clarify that it's related to a database container name rather than the type of memory.